scripts/mkcloud

```diff
@@ -59,6 +59,8 @@ rm -f $mkcconf # we dont want to source old info in the line below
 if (( $want_ipv6 > 0 )); then
     echo ">>> WARNING: IPv6 (want_ipv6) is under active development <<<"
     : ${net_admin:='fd00:0:0:3'}
+    echo 'Enabling want_ipv6 also enables firmware_type="uefi"'
```

Comment:
I don't think we need to warn about this. There's a bunch of want_ options that already behave this way, just set it.

Reply:
done